assets/js/editor/editor.js

```diff
@@ -893,6 +895,19 @@ class Toolbar {
   }
 }
 
+function debounce(func, wait) {
+  let timeout;
+  return (...args) => {
+    const later = () => {
+      clearTimeout(timeout);
+      func(...args);
+    };
+
+    clearTimeout(timeout);
```

Comment:
On the first call `timeout` is undefined. Even if `clearTimeout(…)` doesn't fail, I would prefer `timeout` to be initialized to `null` on line 899 and check for `null` before calling `clearTimeout(…)`.

Reply:
> I think this looks good, but I have a question wrt to the `...args` syntax: is it supported in our targeted browsers?

It should work in our targeted browsers. But due to the fact that currently we don't need the args, I just removed the rest parameter.